roles/hydra-tutorials/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# roles/hydra-tutorials/defaults/main.yml
+
+# the solr version needs to match the default version for [cbeer/solr_wrapper](https://github.com/cbeer/solr_wrapper/blob/master/lib/solr_wrapper.rb#L10-L12) or needs to be explicity set on the local system to match the version here
+  solr_version: 6.1.0
+
```

Comment:
I think it's actually confusing to have to remember different variable names for the solr version depending on how you install it.  That's why I used the same name.  You know what's getting installed based on which role you include.  If you'd rather we name the role something else, I'm up for that, but I think the variable we us to specify which version of Solr gets installed should be the same across any role that installs solr.